### DISCUSSION TITLE
Merge "subproject.viewSummary" and "subproject.viewDetails" into "subproject.view"?

### DISCUSSION THREAD

**Discussion by phpai** [openkfw/TruBudget#867] [Ideas]
Merge "subproject.viewSummary" and "subproject.viewDetails" into "subproject.view" (affects the frontend as well). This makes things like filtering out projects a user is not allowed to see easier and clearer in the code base, and splitting the view intent is not dictated by any requirement at the moment (**or is it??**). Until this is done, filtering subprojects is done by means of the "subproject.viewSummary" intent.

**Comment by mathiashoeld** [comment]
this is already done. @kevinbader please confirm.

**Comment by kevinbader** [comment]
> this is already done. @kevinbader please confirm.

no, this is still valid - there are still two separate _intents_ and endpoints with less-than-crystal-clear semantics attached to them
